graphblas/backend/cuda/spgemm.hpp

```diff
@@ -1,24 +1,16 @@
 #ifndef GRAPHBLAS_BACKEND_CUDA_SPGEMM_HPP_
 #define GRAPHBLAS_BACKEND_CUDA_SPGEMM_HPP_
 
-
-#include "../../../ext/GALATIC/include/dCSR.cuh"
-#include "../../../ext/GALATIC/include/SemiRingInterface.h"
-#include "../../../ext/GALATIC/source/device/Multiply.cuh"
+#include <GALATIC/GALATICMinimumIncludes.cuh>
```

Comment:
Do you mind changing this to `#include "GALATICMinimumIncludes.cuh"`? According to the Google C++ style guide, typically nonsystem headers use the quotation marks instead of angled brackets.

That way, the PROJ_INCLUDES can be `set( PROJ_INCLUDES "./" "ext/moderngpu/include" "ext/GALATIC")`, so it's easier for someone trying to install graphblast to debug for example when PROJ_INCLUDES can't find the GALATIC folder.

Reply:
Whoops, bad inference made me think that was for files that were resolved based on relative path to current file `-I`!